### ISSUE TITLE
Fuse modalities  - (15-1)

### ISSUE THREAD

**Issue by H-Beheiry** [H-Beheiry/Multimodal-Deepfake-Detection-with-XAI#5]
**Fuse modalities using:** 

- [ ]     1.  `y_hat= alpha * logits_video + (1-alpha) * logits_audio`
- [x]     2.  `feature_concat -->> MLP -->> softmax`
- [ ]     3.  (investigate attention maybe) 

**Deadline:** 15-1-26
